### DISCUSSION TITLE
[SOLVED]Can not downgrade directus ?

### DISCUSSION THREAD

**Discussion by korneliusz-w** [directus/directus#8199] [Archive]
Hi

https://docs.directus.io/guides/projects/#upgrading-updating-a-project

1) Fresh installation directus 9.0.0-rc.88
2) Upgrade to rc.93:   npm update, npx directus database migrate:latest

```
13:16:10 ÔťĘ Running migrations...
13:16:10 ÔťĘ Applying Add Required to Fields...
13:16:10 ÔťĘ Applying Update Groups...
13:16:10 ÔťĘ Applying Change Image Metadata Structure...
13:16:10 ÔťĘ Applying Add Geometry Config...
13:16:10 ÔťĘ Applying Remove Limit Column...
13:16:10 ÔťĘ Applying Webhooks Collections Not Null...
13:16:10 ÔťĘ Applying Move Module Setup...
13:16:10 ÔťĘ Database up to date
```

3) Try to downgrade: npm install directus@9.0.0-rc.88 , npx directus database migrate:down

get errors:
```
✨ Running migrations...
Error: Couldn't find migration
    at down (D:\Projekty\directus-wersje\88\node_modules\directus\dist\database\migrations\run.js:65:19)
    at Object.run [as default] (D:\Projekty\directus-wersje\88\node_modules\directus\dist\database\migrations\run.js:36:15)
    at processTicksAndRejections (internal/process/task_queues.js:95:5)
    at async Object.migrate [as default] (D:\Projekty\directus-wersje\88\node_modules\directus\dist\cli\commands\database\migrate.js:13:9)
    at async Command.parseAsync (D:\Projekty\directus-wersje\88\node_modules\commander\lib\command.js:904:5)
```

CMS not workinig
` error select "directus_roles"."id", "directus_roles"."name", "directus_roles"."icon", "directus_roles"."description", "directus_roles"."ip_access", "directus_roles"."enforce_tfa", "directus_roles"."collection_list", "directus_roles"."admin_access", "directus_roles"."app_access", "directus_roles"."module_list" from "directus_roles" where "directus_roles"."id" in ($1) - column directus_roles.module_list does not exist`

**Comment by joselcvarela** [comment]
Hello,
Why do you want to downgrade? We continuously try to push this forward and since we are in a Release Candidate, it is not expected to support downgrades.

**Reply by korneliusz-w** [reply]
Why? After upgrade to 93 custom interfaces stop working and now I have some problem.
https://github.com/directus/directus/discussions/8188

**Reply by korneliusz-w** [reply]
And if it is possible to undo the version in official documentation  why you said you are not supporting it?

**Comment by korneliusz-w** [comment]
I unnecessarily installed an earlier version. Just run migration down on newset version.
